1.1.4 does not build on Nix

Derivation logs:

```
Sourcing python-remove-tests-dir-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing python-remove-bin-bytecode-hook.sh
Sourcing python-imports-check-hook.sh
Using pythonImportsCheckPhase
Sourcing python-namespaces-hook
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/nxqcg590rwak6h2s8xls24xalhrj8qmp-source
source root is source
setting SOURCE_DATE_EPOCH to timestamp 315619200 of file source/umu/umu_util.py
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
patching script interpreter paths in ./configure.sh
./configure.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/0irlcqx2n3qm6b1pc9rsd2i8qpvcccaj-bash-5.2p37/bin/bash"
configure flags: --prefix=/nix/store/9q2i9svkiqc7kf90r14vwvdq3skl7v2x-python3.12-umu-launcher-1.1.4 --prefix=/nix/store/9q2i9svkiqc7kf90r14vwvdq3skl7v2x-python3.12-umu-launcher-1.1.4
:: Created ./Makefile, now run "make" to build.
Running phase: buildPhase
@nix { "action": "setPhase", "phase": "buildPhase" }
build flags: SHELL=/nix/store/0irlcqx2n3qm6b1pc9rsd2i8qpvcccaj-bash-5.2p37/bin/bash PYTHON_INTERPRETER=/nix/store/px2nj16i5gc3d4mnw5l1nclfdxhry61p-python3-3.12.7/bin/python SHELL_INTERPRETER=/run/current-system/sw/bin/bash DESTDIR=/nix/store/9q2i9svkiqc7kf90r14vwvdq3skl7v2x-python3.12-umu-launcher-1.1.4
fatal: not a git repository (or any of the parent directories): .git
:: Building umu 
/nix/store/px2nj16i5gc3d4mnw5l1nclfdxhry61p-python3-3.12.7/bin/python -m build --wheel --skip-dependency-check --no-isolation --outdir=builddir
* Building wheel...
Successfully built umu_launcher-1.1.4-py3-none-any.whl
touch builddir/.build-umu-dist
:: Building umu man pages 
scdoc < docs/umu.1.scd > builddir/umu.1
scdoc < docs/umu.5.scd > builddir/umu.5
touch builddir/.build-umu-docs
:: Building umu-launcher 
sed 's|##INSTALL_PATH##|/nix/store/9q2i9svkiqc7kf90r14vwvdq3skl7v2x-python3.12-umu-launcher-1.1.4/bin/umu-run|g' umu/umu-launcher/umu-run.in > builddir/umu-launcher-run
touch builddir/.build-umu-launcher
:: Building vendored dependencies 
python3 -m pip install urllib3 -t builddir
/nix/store/px2nj16i5gc3d4mnw5l1nclfdxhry61p-python3-3.12.7/bin/python3: No module named pip
make: *** [Makefile.in:104: builddir/.build-umu-vendored] Error 1
```

--- a/packaging/nix/0-Makefile-no-vendor.patch
+++ b/packaging/nix/0-Makefile-no-vendor.patch
@@ -1,0 +1,24 @@
+diff --git a/Makefile.in b/Makefile.in
+index 9324abc..a293409 100644
+--- a/Makefile.in
++++ b/Makefile.in
+@@ -21,7 +21,7 @@ FLATPAK ?= xfalse
+ 
+ .PHONY: all
+ ifeq ($(FLATPAK), xtrue)
+-all: umu-dist umu-launcher umu-vendored
++all: umu-dist umu-launcher
+ endif
+ 
+ .PHONY: install
+@@ -30,8 +30,8 @@ SOURCE_DATE_EPOCH = $(shell LC_ALL=C date --date='@1580601600')
+ all: zipapp
+ install: zipapp-install
+ else
+-all: umu-dist umu-docs umu-launcher umu-vendored
+-install: umu-install umu-launcher-install umu-vendored-install
++all: umu-dist umu-docs umu-launcher
++install: umu-install umu-launcher-install
+ endif
+ 
+ 

--- a/packaging/nix/umu-launcher.nix
+++ b/packaging/nix/umu-launcher.nix
@@ -3,6 +3,7 @@ python3Packages.buildPythonPackage {
   pname = "umu-launcher";
   version = "${version}";
   src = umu-launcher;
+  patches = [ ./0-Makefile-no-vendor.patch ];
   pyproject = false;
   depsBuildBuild = [
     pkgs.meson
@@ -18,6 +19,7 @@ python3Packages.buildPythonPackage {
     pkgs.bubblewrap
     pkgs.python3Packages.xlib
     pkgs.python3Packages.filelock
+    pkgs.python3Packages.urllib3
   ];
   makeFlags = [ "PYTHON_INTERPRETER=${pyth1}/bin/python" "SHELL_INTERPRETER=/run/current-system/sw/bin/bash" "DESTDIR=${placeholder "out"}" ];
   dontUseMesonConfigure = true;

